deploy/charts/harvester/values.yaml

```diff
@@ -462,12 +462,12 @@ harvester-load-balancer:
   image:
     imagePullPolicy: IfNotPresent
     repository: rancher/harvester-load-balancer
-    tag: v0.3.0
+    tag: v0.4.3
```

Comment:
Isn't 0.4.x used for v1.4 branch?

Reply:
I had the same question initially, but after discussing offline with @starbops, it seems that this is the desired version...

Reply:
According to https://github.com/harvester/harvester/issues/6590#issuecomment-2527861060, v0.4.3 is the desired version to bump to. cc @w13915984028